### DISCUSSION TITLE
Repo for community plugins, fields, tools...

### DISCUSSION THREAD

**Discussion by Yakoust** [keystonejs/keystone#5317] [Feature requests]
Hi all,

My idea, inspired by other frameworks (eg https://github.com/capacitor-community/), would be to create a specific repo for tools, fields and plugins created by developers who work around KeystoneJS and who want share them and submit them to the community.

For example, I developed a text field (based on the existing component) with a size limit, a parameterized text field, or even a GCP connector for storing uploaded files. I will be happy to share this with other developers who have the same needs.

**Comment by gautamsi** [comment]
@Yakoust I am maintainer of https://github.com/keystonejs-contrib/keystonejs-contrib which hosts some community packages. It is going to host more packages as some of them are being dropped from `keystone-next`.

if you have something and you want to submit there, it will be great.

**Reply by timleslie** [reply]
@Yakoust This is a fantastic idea, and @gautamsi has a great repo for hosting these kinds of packages. Developing new features as a contrib package is a great way to share your ideas and benefit from the knowledge of others 🚀
